src/interface/viewport.h

```diff
@@ -41,7 +41,7 @@ enum {
 	VIEWPORT_FLAG_HIDE_BASE = (1 << 12),
 	VIEWPORT_FLAG_HIDE_VERTICAL = (1 << 13),
 	VIEWPORT_FLAG_INVISIBLE_SPRITES = (1 << 14),
-	VIEWPORT_FLAG_15 = (1 << 15)
+	VIEWPORT_FLAG_HIDE_TOP_BOTTOM_TOOLBAR = (1 << 15),
```

Comment:
@Gymnasiast I didn't find a place where this viewport was used. (Did a grep search on all source files)

@IntelOrca I create a new commit, without a variable to store the status of the show/hide toggle. Maybe it is not necessary at all to create a flag or var if we are only hiding the top/bottom toolbar with a shortcut.